### PR TITLE
Make function names in tasks intended to be copy pasted styled like code.

### PR DESCRIPTION
**Pull request by Fejiberglibstein**
My general idea was to make only class names and function names have styling, and to only style the first reference to it.
Already made class names and function names will *not* have styling, since students will get autocomplete for them in Intellij.

### Preview
image
image

### CODE REVIEW

File: templates/hw/task1.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h3>
         </p>
 
         <ol>
-            <li>average
+            <li><Code>average</Code>
```
Comment (zaida04) [inline templates/hw/task1.html:65]: Are these supposed to be capitalized? Not sure if this indicates some sort of custom element, but the native HTML code element is <code>.

Comment (Fejiberglibstein) [reply templates/hw/task1.html:65]: Oh yea you're right. I initially had it lower case but when I had to redo the hws I guess I forgot

Comment (Fejiberglibstein) [reply templates/hw/task1.html]: Fixed, thanks for the catch!

**Review by zaida04 — APPROVED** [review]
lgtm, dark mode and light mode looks fine

IMG_0330

**Pull request merged**